### PR TITLE
Add 1.26 Bottlerocket e2e tests

### PR DESCRIPTION
**Pull request by taneyland**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/e2e/tinkerbell_test.go [hunk 1]
```diff
@@ -54,12 +54,12 @@ func TestTinkerbellKubernetes123BottleRocketAWSIamAuth(t *testing.T) {
 	runTinkerbellAWSIamAuthFlow(test)
 }
 
-func TestTinkerbellKubernetes125BottleRocketAWSIamAuth(t *testing.T) {
+func TestTinkerbellKubernetes126BottleRocketAWSIamAuth(t *testing.T) {
```
Comment (cxbrowne1207) [inline test/e2e/tinkerbell_test.go:57]: Do we want none of tests for 125? I noticed that there is currently still one for 123

Comment (taneyland) [reply test/e2e/tinkerbell_test.go:57]: There isn't really a set protocol. We update most of the tests to use the latest version, but we also leave a few tests with older versions just to have insight and coverage of the behavior with other versions, just to ensure that we aren't introducing breaking changes for older versions.

**Pull request merged**